.github/actions/setup-deps/action.yaml

```diff
@@ -76,6 +76,8 @@ inputs:
     default: 'pytest-cov<=2.10.1'
   pytest-xdist:
     default: 'pytest-xdist'
+  pyedr:
```

Comment:
This also needs adding to the azure pipelines file: https://github.com/MDAnalysis/mdanalysis/blob/2c32ed11c1e6b800397f147e4242f7325900b669/azure-pipelines.yml#L84-L98

Reply:
We are not at this point making pyedr a core dependency - going by the fact that we aren't touching setup.py etc... (I would like to get more coredev oks on this anyways)

As a result, I would ask that this be moved to the pip optional dependencies section.